agimus_controller_ros/agimus_controller_ros/agimus_controller_parameters.yaml

```diff
@@ -80,3 +80,19 @@ agimus_controller_params:
     type: bool
     default_value: True
     description: "Wether we account for self-collision"
+  collision_pairs_names:
+    type: string_array
+    default_value: [""]
+    description: List of names of collision pairs passed to the OCP.
+    read_only: true
+  __map_collision_pairs_names:
```

Comment:
How/when is this one used? I did not see it before so I'm curious. Maybe would be nice to have some notes on the effect of this for people like me

Reply:
Documentation is here (read the.... README ;) )
https://github.com/PickNikRobotics/generate_parameter_library

Reply:
@MaximilienNaveau for the part that I do not get is how it knows how to break down a list of strings to "first" and "second"

Reply:
Nevermind, I think I get it now! It is not automatic, we define it ourselves in another .yaml file

Reply:
Yes this is a template for automatic code generation.
Hence during the code generation it transform this:
```yaml
collision_pairs: ["pair1", "pair2"]
pair1:
  first: "banana"
  second: "bread"
pair2:
  first: "pinky"
  second: "pie"
```
Into a dictionary.

Is that clear ?